admin/create-dumps.sh

```diff
@@ -63,11 +63,13 @@ trap finish EXIT
 if [ "$DUMP_TYPE" == "full" ]; then
     if ! /usr/local/bin/python manage.py dump create_full -l "$TMPDIR" -t $DUMP_THREADS --last-dump-id; then
 	echo "Full dump failed, exiting!"
+        finish
```

Comment:
the `trap finish EXIT` above will cause `finish` to call any time the script exits, even when it's finished with `exit 1`. These additions aren't needed

Reply:
Then how is it possible that this script leaves a dirty dump around when it fails? That is the whole point of this exercise -- how can a failed dump be let after this script terminates. do you see any way?